src/DotNetLightning.Core/Utils/Extensions.fs

```diff
@@ -148,11 +148,10 @@ type System.Collections.BitArray with
         BitArray.From5BitEncoding(b)
         
     member this.Reverse() =
-        let length = this.Length
-        let mutable result = Array.zeroCreate length
-        for i in 0 .. (length - 1) do
-            result.[i] <- this.[length - i - 1]
-        result
+        let boolArray: array<bool> = Array.ofSeq (Seq.cast this)
+        Array.Reverse boolArray
```

Comment:
there's no need to hardcode `Array` here, `.Reverse` exists in System.Linq and can be applied over IEnumerable

Reply:
Unfortunately `BitArray` can't be constructed from a sequence (as far as I could tell), so I still need to explicitly convert to an `Array`. So whether I call `Array.Reverse` or `Linq.Enumerable.Reverse` is kinda inconsequential.